mite/context.py

```diff
@@ -50,9 +50,7 @@ def config(self):
 
     @property
     def should_stop(self):
-        if self._should_stop_func is not None:
-            return self._should_stop_func()
-        return False
+        return False if self._should_stop_func is None else self._should_stop_func()
```

Comment:
`self._should_stop_func and self._should_stop_func() or False` might be an alternative